ui/page/send/page.go

```diff
@@ -554,6 +597,29 @@ func (pg *Page) HandleUserInteractions() {
 	}
 }
 
+// Handle is called just before Layout() to determine if any user
+// interaction recently occurred on the modal and may be used to update the
+// page's UI components shortly before they are displayed.
+func (pg *Page) Handle() {
```

Comment:
```suggestion
// Handle is like HandleUserInteractions but Handle is called if this page is displayed as a modal
// while HandleUserInteractions is called if this page is displayed as a full page. Either Handle or
// HandleUserInteractions will be called just before Layout() is called to determine if any user
// interaction recently occurred on the modal or page and may be used to update any affected
// UI components shortly before they are displayed by the Layout() method.
func (pg *Page) Handle() {
```

Reply:
OnResume and OnDismiss need similar comments.

Reply:
OnResume and OnDismiss need similar comments.

Reply:
Your comment is not where it is needed that why its not resolved.